setup.cfg

```diff
@@ -19,6 +19,6 @@ line_length = 79
 multi_line_output = 0
 known_standard_library = pkg_resources,setuptools
 known_first_party = mmaction
-known_third_party = cv2,joblib,matplotlib,mmcv,numpy,pandas,pytest,scipy,seaborn,torch
+known_third_party = cv2,joblib,matplotlib,mmcv,mmdet,numpy,pandas,pytest,scipy,seaborn,torch
```

Comment:
is it possible to make mmdet an optional dependency?

Reply:
We can. But I think make it compulsory is also OK. Since mmdet is not heavy, the wheel of mmdet is only 490KB, and there are few additional packages required by mmdet (only mmpycocotools and terminaltables) on my machine.

Reply:
Nonono, mmdet is not easy to install.

Reply:
I can install mmdet with one command `pip install mmdet` on my machine. Why it is difficult to install?

Reply:
Good question. No difficulty for super users.

![image](https://user-images.githubusercontent.com/9464825/102160673-d9951080-3ec0-11eb-9e37-eb29658213b4.png)

Reply:
Would you please help me chech that if you can install mmdet with `pip install mmdet` on your machine? 
I think `pip install` is much easier than installing from src code (if so you need to compile ops). Since we only use some features of mmdet and do not focus on developing detection algorithms, `pip install` is enough.

Reply:
![image](https://user-images.githubusercontent.com/22089207/102166934-41028e80-3ec8-11eb-828d-c819d90bbfb4.png)

Reply:
A few days ago(when you created this PR), I told myself to remind you: We should make mmdetection as an optional dependency... :)

Reply:
Fine, it's also OK to make it optional.